.projenrc.js

```diff
@@ -42,18 +42,12 @@ const project = new AwsCdkTypeScriptApp({
   context: {
     //vpc_tag_name: 'ecsworkshop-base/BaseVPC', // TAG Name of the VPC to create the cluster into (or 'default' or remove to create new one)
     enablePrivateLink: 'false', // this parameter seems to works only one
-    useEFS: false,
-
-    //os_domain: 'magento-cdk4', // default to $CDK_STACK_NAME
-    os_master_user_name: 'magento-master-os',
-
-    //db_name: 'magento3', // default to env $CDK_STACK_NAME
-    db_user: 'magentodbuser',
+    useEFS: 'yes',
```

Comment:
Please add comment on what this parameter does

Reply:
It is done in the readme, you want also comments in this file ?

Reply:
This should be ok.